lib/loader/umd/index.js

```diff
@@ -46,6 +46,7 @@ var loader = (function(exports) {
   const THIS = Symbol();
   const STRING_DECODE_THRESHOLD = 32;
   const decoder = new TextDecoder("utf-16le");
+  const rootsSym = Symbol();
```

Comment:
awesome use of the newish symbol type. this works very fast in memory compared to leverage a map as a dict.